src/input.ts

```diff
@@ -44,6 +44,10 @@ export type InputFormat =
   | EpubOpfInput;
 
 export function detectInputFormat(entry: string): InputFormat {
+  if (/^https?:\/\//.test(entry)) {
+    return { format: 'html', entry };
```

Comment:
現在は `SingleHtmlInput` と `WebbookInput` を設定する機構がないため、デフォルトで `WebbookInput` になる設定でお願いします。

```suggestion
    return { format: 'webbook', entry };
```